TTS/vocoder/models/gan.py

```diff
@@ -198,7 +198,7 @@ def _log(name: str, ap: AudioProcessor, batch: Dict, outputs: Dict) -> Tuple[Dic
         Returns:
             Tuple[Dict, Dict]: log figures and audio samples.
         """
-        y_hat = outputs[0]["model_outputs"]
+        y_hat = outputs[0]["model_outputs"] if outputs[0] is not None else outputs[1]["model_outputs"]
```

Comment:
I guess the condition we need to check `self.train_disc` is right? The only case there are no outputs[0] is when `self.train_disc` is False.

Reply:
updated